docs/source/markdown/podman-build.1.md

```diff
@@ -353,11 +365,15 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
 **--net**, **--network**=*string*
 
 Sets the configuration for network namespaces when handling `RUN` instructions.
-The configured value can be "" (the empty string) or "container" to indicate
-that a new network namespace should be created, or it can be "host" to indicate
-that the network namespace in which `podman` itself is being run should be
-reused, or it can be the path to a network namespace which is already in use by
-another process.
+
+Valid _mode_ values are:
+
+Valid _mode_ values are:
```

Comment:
cut/paste error?  Line 369 and 371 look to be dupes

Reply:
Fixed